server/services/matches.ts

```diff
@@ -62,4 +62,26 @@ const createMatchService = async (
   };
 };
 
-export default createMatchService;
+export const getAllMatches = async () => {
```

Comment:
Add the returning type for the function getAllMatches Promise<matchesInterface> so that you want to create matchesInterface to resolve this issue